Package.swift

```diff
@@ -1,14 +1,30 @@
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the DeviceKit open source project
-//
-// Copyright © 2014 - 2018 Dennis Weissmann and the DeviceKit project authors
-//
-// License: https://github.com/dennisweissmann/DeviceKit/blob/master/LICENSE
-// Contributors: https://github.com/dennisweissmann/DeviceKit#contributors
-//
-//===----------------------------------------------------------------------===//
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
```

Comment:
Please add back the copyright message though:

```suggestion
// The swift-tools-version declares the minimum version of Swift required to build this package.

//===----------------------------------------------------------------------===//
//
// This source file is part of the DeviceKit open source project
//
// Copyright © 2014 - 2018 Dennis Weissmann and the DeviceKit project authors
//
// License: https://github.com/dennisweissmann/DeviceKit/blob/master/LICENSE
// Contributors: https://github.com/dennisweissmann/DeviceKit#contributors
//
//===----------------------------------------------------------------------===//
```